### DISCUSSION TITLE
Pilot remote pilot logging - hackathon 3.08.2023

### DISCUSSION THREAD

**Discussion by martynia** [DIRACGrid/DIRAC#7159] [Q&A]
Hello,
I tried to use a simple client to send a test log to Tornado server on the certification machine. After setting the `LoggingPlugin` and `UploadPath` I could contact the server, but was getting 401. This seemed to be caused by an option in `Systems/WorkloadManagement/Certification/Services/TornadoPilotLogging/Authorization`. The option(s) are not taken into account. `sendMessage` should work with an `Operator` property, but the logs complain that a user `(dirac_admin)` is not an `Operator` (it clearly is). This part of the workflow works if restrictions are waived (i.e the option is removed).

Summary: The test shows that a message shipment works with a basic auth.

**Comment by fstagni** [comment]
This is because in ConfigTemplate (https://github.com/DIRACGrid/DIRAC/blob/integration/src/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg#L29) you added things like `sendMessage = "Operator"` while it should be `sendMessage = Operator` (for all of them)

**Reply by martynia** [reply]
I changed the config on the server (removed the quotes). I restarted Tornado, no change.

**Reply by fstagni** [reply]
There was a misconfiguration, which I think I fixed. Try again.

**Reply by martynia** [reply]
OK, it works now with a dirac_admin proxy. If we could submit jobs we could test a real pilot log. I'm off today, we can come back to this on Monday.
Thanks, JM
